hack/loadtest/test.sh

```diff
@@ -0,0 +1,44 @@
+set -e
+
+# Functions that given a number it creates a namespace
+# and in that namespace it creates a monitoring stack
+function create_monitoring_stack() {
+    stack_number=$1
+    export MS_NAME=stack-$stack_number
+    export NAMESPACE=loadtest-$stack_number
+
+    monitoring_stack="apiVersion: monitoring.rhobs/v1alpha1
```

Comment:
it may be better to read from the samples, or a test-data directory.

Reply:
I think it is better to use heredocs instead e.g. 

read -r -d '' VAR <<- EO_YAML
     apiVersion: 
     kind:
     metadata: ...
EO_YAML

Reply:
From StackOverflow:
> I think it's worth mentioning that if you have set -o errexit (a.k.a set -e) in your script and you use this then it will terminate your script because read returns a non-zero return code when it reaches EOF.

Still, I've updated it to use heredoc